BE/src/main/resources/schema.sql

```diff
@@ -0,0 +1,86 @@
+DROP TABLE IF EXISTS member;
+CREATE TABLE member
+(
+    member_id BIGINT AUTO_INCREMENT NOT NULL,
+    email     VARCHAR(255) NOT NULL,
+    password  VARCHAR(255) NOT NULL,
+    PRIMARY KEY (member_id)
+);
+
+DROP TABLE IF EXISTS order_sheet;
+CREATE TABLE order_sheet
+(
+    order_id    BIGINT AUTO_INCREMENT NOT NULL,
+    banchan_id  BIGINT NOT NULL,
+    member_id   BIGINT NOT NULL,
+    quantity    INTEGER NOT NULL,
+    total_price INTEGER NOT NULL,
+    PRIMARY KEY (order_id),
+    FOREIGN KEY (banchan_id) REFERENCES banchans (banchan_id),
+    FOREIGN KEY (member_id)  REFERENCES member (member_id)
+);
+
+DROP TABLE IF EXISTS banchans;
+CREATE TABLE banchans
+(
+     banchan_id   BIGINT AUTO_INCREMENT NOT NULL,
+     category_id  INTEGER NOT NULL,
+     banchan_name VARCHAR(255) NOT NULL,
+     description  VARCHAR(255) NOT NULL,
+     or_price     INTEGER NOT NULL,
+     dc_price     INTEGER NOT NULL,
+     stock        INTEGER NOT NULL,
+     label_id     INTEGER NOT NULL,
+     main_image   VARCHAR(255) NOT NULL,
+     delivery_id  INTEGER NOT NULL,
+     best_id      BIGINT,
```

Comment:
ask; 이건 어떤 필드인가요? 아래처럼 코멘트 추가하는 것도 좋겠습니다.

```suggestion
     best_id      BIGINT COMMENT '이렇게 코멘트 남길 수 있습니다.',
```

Reply:
`best_id`는 기획전에서 가장 많이 셀링된 best 음식을 식별하기위한 id 값입니다.